### PR TITLE
BETA: Register rockstarbuddies.is-a.dev

### PR DESCRIPTION
**Pull request by RockstarBuddies**
Added `rockstarbuddies.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by RockstarBuddies** [comment]
Approximately how much time does it take to get reviewed?

File: domains/adityagupta.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "RockstarBuddies",
+       "email": "yourrockingbuddy@gmail.com",
+       "discord": "899663426970865674"
+    },
+
+    "record": {
+        "CNAME": "blog"
```
Comment (WilliamDavidHarrison) [inline domains/adityagupta.json:9]: Please use a valid record here.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/adityagupta.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "RockstarBuddies",
+       "email": "yourrockingbuddy@gmail.com",
+       "discord": "899663426970865674"
+    },
+
+    "record": {
+        "CNAME": "rockstarbuddies"
```
Comment (WilliamDavidHarrison) [inline domains/adityagupta.json:9]: Please use a valid record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/adityagupta.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "RockstarBuddies",
+       "email": "yourrockingbuddy@gmail.com",
+       "discord": "899663426970865674"
+    },
+
+    "record": {
+        "CNAME": "2405:201:4003:4a4b::c0a8:1d01"
```
Comment (WilliamDavidHarrison) [inline domains/adityagupta.json:9]: We do not support AAAA records. Also, that is an invalid CNAME record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/adityagupta.json (hunk 3 above)
Comment (RockstarBuddies) [reply domains/adityagupta.json]: is this file even needed? I filed for "rockstarbuddies" one

**Review by wdhdev — APPROVED** [review]

**Pull request merged**